packages/plugins/users-permissions/admin/src/permissions.js

```diff
@@ -6,7 +6,7 @@ const pluginPermissions = {
   ],
   createRole: [{ action: 'plugin::users-permissions.roles.create', subject: null }],
   deleteRole: [{ action: 'plugin::users-permissions.roles.delete', subject: null }],
-  readRoles: [{ action: 'plugin::users-permissions.roles.read', subject: null }],
+  readRole: [{ action: 'plugin::users-permissions.roles.read', subject: null }],
```

Comment:
I think this should be kept in the plural form as it is the permission to read all roles

Reply:
oops, updated